tools/build-vs2015.cmd

```diff
@@ -0,0 +1,6 @@
+set "VS_TOOLS_VERSION=vs2015"
+REM Ref. https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2014%202015.html
+set "CMAKE_GEN=Visual Studio 14 2015 Win64"
```

Comment:
Are we going to support 32-bit Windows?

Reply:
Yes. I can add a parameter for that, with default being Win64.